components/TimeoutModal.tsx

```diff
@@ -5,76 +5,71 @@ import Modal from 'react-bootstrap/Modal'
 import { Button } from './Button'
 import getUrl from '../utils/getUrl'
 
-let timeOutTimerId: NodeJS.Timeout | null = null
 let warningTimerId: NodeJS.Timeout | null = null
 
 export interface TimeoutModalProps {
```

Comment:
Somewhere in this file, maybe above this line, it seems useful to provide a short description of the expected behavior of this timeout modal? "In order to match EDD's session expiration policy, we redirect users back to EDD login page after REDIRECT_TIMER minutes. We also show this timeout modal WARNING_DURATION minutes prior to the redirect. X'ing out the modal or clicking outside it lets the user cancel the redirect and stay on the page."